### PR TITLE
Enhancement: toolbar and header visual changes

### LINKED ISSUE #3040: [Enhancement] Toolbar and header visual changes
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
UX has made some small visual changes to the toolbar on mobile + desktop (before and after scroll):

- [x] Toolbar on mobile should always be xxl/56px (effectively reverts changes done in #2219)
- [x] Toolbar on desktop should always be xxxxxl/72px (new size should be added to sizes-ramp)
- [x] Toolbar separator bottom-line should always be visible on desktop
   - [x] Except when [tab-navigation](https://cookbook.kirby.design/#/examples/page/tab-navigation) is used 
- [x] Toolbar background should be Kirby black + 1% transparent
   - [ ]  Please note: the design specs shows a translucent toolbar, but as this has implications when using tab navigation, this change is postponed to #3247
- [x] ~~Toolbar buttons should have 16px spacing on the right on mobile and 24px on desktop~~
       _**Please note:** This enhancement will be implemented in a separate issue when we fix the spacing of toolbar action buttons and space available for toolbar title (is currently partly overlapped by toolbar buttons)_ 

 

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

### PR DESCRIPTION
**Pull request by jakobe**
## Which issue does this PR close?

This PR closes #3040

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [ ] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [ ] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: libs/designsystem/page/src/page.component.spec.ts [hunk 1]
```diff
@@ -92,446 +95,794 @@ describe('PageComponent', () => {
...
-      expect(pageSubtitle).toHaveComputedStyle({
-        'margin-left': '0px',
-        'margin-right': '0px',
-        'margin-top': size('xxs'),
-        'margin-bottom': '0px',
-        'padding-left': '0px',
-        'padding-right': '0px',
-        'padding-top': '0px',
-        'padding-bottom': '0px',
+      it('should render subitle with correct margin and padding', () => {
```
Comment (mark-drastrup) [inline libs/designsystem/page/src/page.component.spec.ts]:
```suggestion
      it('should render subtitle with correct margin and padding', () => {
```

File: libs/designsystem/page/src/page.component.spec.ts [hunk 2]
```diff
@@ -92,446 +95,794 @@ describe('PageComponent', () => {
...
+        await TestHelper.whenReady(ionContent);
+        const pageSubtitle = ionContent.querySelector('.page-subtitle');
 
-      expect(spectator.component.subtitle).toEqual(subtitleText);
-      expect(pageSubtitle).toHaveText(subtitleText, true);
+        expect(spectator.component.subtitle).toEqual(subtitleText);
+        expect(pageSubtitle).toHaveText(subtitleText, true);
+      });
+
+      it('should render subitle with correct margin and padding', async () => {
```
Comment (mark-drastrup) [inline libs/designsystem/page/src/page.component.spec.ts]:
```suggestion
      it('should render subtitle with correct margin and padding', async () => {
```

File: libs/designsystem/page/src/page.component.spec.ts [hunk 3]
```diff
@@ -92,446 +95,794 @@ describe('PageComponent', () => {
...
+          spectator.detectChanges();
+        });
 
-      expect(defaultHandler).toHaveBeenCalledTimes(1);
-    });
+        it('should apply correct content width', async () => {
+          await TestHelper.whenReady(ionContent);
+          const contentInner = ionContent.querySelector('.content-inner');
+          expect(contentInner).toHaveComputedStyle({
+            'max-width': '792px',
```
Comment (mark-drastrup) [inline libs/designsystem/page/src/page.component.spec.ts:590]: Same as above - can we use the DesignTokenHelper to get this value?

File: libs/designsystem/page/src/page.component.spec.ts [hunk 4]
```diff
@@ -92,446 +95,794 @@ describe('PageComponent', () => {
...
 
-    it('should call the default click handler if no back-button-click observer is provided', () => {
-      const defaultHandler = jasmine.createSpy();
-      ionBackButton.onclick = defaultHandler;
+    describe('with maxWidth is defined', () => {
+      it('should apply the correct content width', async () => {
+        await TestHelper.whenReady(ionContent);
+        const contentInner = ionContent.querySelector('.content-inner');
+        expect(contentInner).toHaveComputedStyle({
+          'max-width': '720px',
```
Comment (mark-drastrup) [inline libs/designsystem/page/src/page.component.spec.ts:576]: Can we use the DesignTokenHelper to get this value?

File: libs/designsystem/page/src/page.component.spec.ts [hunk 5]
```diff
@@ -92,446 +95,794 @@ describe('PageComponent', () => {
...
+          spectator.component.maxWidth = 'optimized';
+          spectator.detectChanges();
+        });
 
-      spectator.click(ionBackButton);
+        it('should apply correct content width', async () => {
+          await TestHelper.whenReady(ionContent);
+          const contentInner = ionContent.querySelector('.content-inner');
+          expect(contentInner).toHaveComputedStyle({
+            'max-width': '1092px',
```
Comment (mark-drastrup) [inline libs/designsystem/page/src/page.component.spec.ts:605]: Same as above - can we use the DesignTokenHelper to get this value?

File: libs/designsystem/tab-navigation/src/tab-navigation/tab-navigation.component.scss [hunk 6]
```diff
@@ -1,11 +1,35 @@
...
+  }
+
+  // Divider
+  &::before {
+    content: '';
+    background-color: $border-color-standard;
+    position: absolute;
+    height: $bottom-border-height;
+    bottom: 0;
+    z-index: 1;
```
Comment (mark-drastrup) [inline libs/designsystem/tab-navigation/src/tab-navigation/tab-navigation.component.scss]: This value can also be set by using `utils.z('default')`

**Review by mark-drastrup — COMMENTED** [review]
Great job @jakobe! I only have some minor comments :)

File: libs/designsystem/page/src/page.component.spec.ts (hunk 3 above)
Comment (jakobe) [reply libs/designsystem/page/src/page.component.spec.ts:590]: Unfortunately not as they are magic values hardcoded into page.component.scss :(

File: libs/designsystem/page/src/page.component.spec.ts (hunk 5 above)
Comment (jakobe) [reply libs/designsystem/page/src/page.component.spec.ts:605]: Unfortunately not as they are magic values hardcoded into page.component.scss :(

File: libs/designsystem/tab-navigation/src/tab-navigation/tab-navigation.component.scss (hunk 6 above)
Comment (jakobe) [reply libs/designsystem/tab-navigation/src/tab-navigation/tab-navigation.component.scss]: @mark-drastrup Good catch! 👌🏻

**Review by mark-drastrup — APPROVED** [review]

**Pull request merged**